google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditor.java

```diff
@@ -108,12 +111,20 @@
   private JButton moduleSettingsButton;
   private JCheckBox hiddenValidationTrigger;
   private JLabel noSupportedModulesWarning;
+  private JLabel environmentLabel;
 
   private static final boolean PROMOTE_DEFAULT = false;
   private static final boolean STOP_PREVIOUS_VERSION_DEFAULT = false;
 
   public AppEngineFlexibleDeploymentEditor(Project project, AppEngineDeployable deploymentSource) {
     this.deploymentSource = deploymentSource;
+
+    gcpProjectSelector = appEngineConfig.getProjectSelector();
```

Comment:
yes this is hideous; i'm already working on the next PR where I plan to refactor this. Likely I will not reassign to local members and just get them via the getters.

If you're ok keeping it here, it will shortly be fixed in the next PR